src/models/UserMapper.php

```diff
@@ -174,6 +196,10 @@ public function transformResults($results, $verbose)
                     . $row['ID'] . '/hosted/';
                 $list[$key]['talk_comments_uri'] = $base . '/' . $version . '/users/'
                     . $row['ID'] . '/talk_comments/';
+                
+                if ($verbose && isset($this->_request->user_id)) {
```

Comment:
$user_id is a public property of Request, so it always exists. It's either null or a number.